app/src/styles.css

```diff
@@ -56,7 +56,7 @@ html {
 body {
   padding: 0;
   margin: calc((100vh / 25) * 1.563) calc((100vw / 25) * 1.563);
-  background-color: white;
+  background-color: #a8d0e6;
```

Comment:
Can't remember material UI as much but do these belong in the theme file?

As we discussed, I would save most styling for demo prep but up to you!